pkg/proxy/proxy.go

```diff
@@ -57,11 +62,11 @@ func NewProxy(cfg *Config) *Proxy {
 	transport := &http.Transport{
 		Proxy: http.ProxyFromEnvironment,
 		Dial: (&net.Dialer{
```

Comment:
It looks like `Dial` is now deprecated in favor of `DialContext`. Should we switch?

https://pkg.go.dev/net/http@go1.20.3#Transport

Reply:
@spadgett yeah lets switch to the new one 👍